data/json/recipes/armor/feet.json

```diff
@@ -422,6 +422,17 @@
     "qualities": [ { "id": "KNIT", "level": 1 } ],
     "components": [ [ [ "yarn", 75 ] ] ]
   },
+  {
+    "result": "socks_ankle",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "time": "8 m",
+    "autolearn": true,
+    "using": [ [ "sewing_standard", 4 ] ],
+    "components": [ [ [ "rag", 1 ] ] ]
```

Comment:
8 minutes feels like way too fast for a pair of socks. Also, 1 rag is too low. There should be at least 2 rags.

Reply:
Basic socks take 10m to craft, these are smaller so I made them take a little bit less time. Basic socks also take 2 rags, but I agree, ankle socks should take 2 rags too. Maybe less thread?

Reply:
Seems @Rail-Runner should release the next part of the tailoring re-work then to fix the sock making time issue.